tests/client/api_v1/test_get_balance.c

```diff
@@ -43,17 +63,32 @@ void test_get_balance() {
   res = res_balance_new();
   TEST_ASSERT_NOT_NULL(res);
 
-  // test for success
-  TEST_ASSERT_EQUAL_INT(0, get_balance(&conf, addr_hex, res));
+  // test for success - ED25519 address
+  TEST_ASSERT_EQUAL_INT(0, get_balance(&conf, ED25519, addr_hex_ed25519, res));
   if (res->is_error == true) {
     printf("Error: %s\n", res->u.error->msg);
   } else {
     // validate address type
     TEST_ASSERT(ADDRESS_VER_ED25519 == res->u.output_balance->address_type);
     // validate address data
-    TEST_ASSERT_EQUAL_MEMORY(addr_hex, res->u.output_balance->address, IOTA_ADDRESS_HEX_BYTES);
+    TEST_ASSERT_EQUAL_MEMORY(addr_hex_ed25519, res->u.output_balance->address, IOTA_ADDRESS_HEX_BYTES);
   }
 
+  // test for success - BECH32 address
+  TEST_ASSERT_EQUAL_INT(0, get_balance(&conf, BECH32, addr_hex_bech32, res));
+  if (res->is_error == true) {
+    printf("Error: %s\n", res->u.error->msg);
+  } else {
+    // Note : Bech32 also returns ED25519 address in the response
+    // validate address type
+    TEST_ASSERT(ADDRESS_VER_ED25519 == res->u.output_balance->address_type);
+    char addr_hex_str[IOTA_ADDRESS_HEX_BYTES + 1] = {0};
+    TEST_ASSERT(address_bech32_to_hex("iota", addr_hex_bech32, addr_hex_str, sizeof(addr_hex_str)) == 0);
+    // Converting hex string to lower case to check equality
+    for (int i = 0; i < IOTA_ADDRESS_HEX_BYTES; i++) addr_hex_str[i] = tolower(addr_hex_str[i]);
```

Comment:
coding style, curly braces
```suggestion
    for (int i = 0; i < IOTA_ADDRESS_HEX_BYTES; i++) {
      addr_hex_str[i] = tolower(addr_hex_str[i]);
    }
```

Reply:
Added curly braces.